pulsar-broker/src/test/java/org/apache/pulsar/client/impl/TransactionEndToEndTest.java

```diff
@@ -811,7 +811,7 @@ private void txnCumulativeAckTest(boolean batchEnable, int maxBatchSize, Subscri
     public Transaction getTxn() throws Exception {
         return pulsarClient
                 .newTransaction()
-                .withTransactionTimeout(10, TimeUnit.SECONDS)
+                .withTransactionTimeout(10, TimeUnit.MINUTES)
```

Comment:
Two questions:
- Can we reduce the number of messages to send instead of increasing the timeout of TXN?
- I found that the test failed due to a timeout of 30 seconds. Since we set 10s timeout for TXN, why this transaction is not failed by transaction timeout?

Reply:
Please use a more reasonable value than 10 minutes...

Reply:
The reason I used a 10-minute timeout is also to prevent any overlooked transactions from affecting other tests after they time out. When I was reproducing the issue locally, the `testTxnTimeoutAtTransactionMetadataStore` often failed. After analyzing, it was found that some transactions timed out during the test run. Therefore, I set the timeout to 10 minutes to avoid the possibility that other tests creating transactions might affect these types of tests.

Reply:
>I found that the test failed due to a timeout of 30 seconds. Since we set 10s timeout for TXN, why this transaction is not failed by transaction timeout?
@poorbarcode In the logs of the issue, it can be found that this test failed due to the transaction timeout.

Reply:
one minute is enough

Reply:
Reach Reach a consensus to prevent transactions between tests from affecting each other, use ten minutes.